scrapers/reddit.js

```diff
@@ -0,0 +1,56 @@
+const BaseLoklakScrapper = require('./base');
+
+class RedditLoklakScrapper extends BaseLoklakScrapper {
+
+    constructor() {
+        super('Reddit', 'https://www.reddit.com');
+    }
+
+    argumentSanityCheck(args) {
+        super.argumentSanityCheck(args);
+
+        if (args.length <= 2) {
+            console.error('Atleast one argument required.');
+            process.exit(-1);
+        }
+
+        return true;
+    }
+
+    onInit() {
+        this.REQUEST_URL = this.BASE_URL + "/r/" + this.PROC_ARGS[2];
```

Comment:
@djmgit, using `this.SLICED_PROC_ARGS[0]` instead of `PROC_ARGS` would be better